src/sort/simple-sort-directive.js

```diff
@@ -154,6 +154,26 @@ angular.module('patternfly.sort').directive('pfSimpleSort', function () {
     },
     templateUrl: 'sort/simple-sort.html',
     controller: function ($scope) {
+      var setSortDirectionTooltip = function () {
+        var tooltip = '';
+
+        if ($scope.config.currentField.sortType === 'numeric') {
+          if ($scope.config.isAscending) {
+            tooltip = 'Sort numerically, from least to greatest';
```

Comment:
When I hover over a button, I expect the tooltip to tell me what pressing the button would do, not what the current state of the button is.  Here we are doing the latter and I just want to make sure that is intentional.